File: .claude/skills/cli-builder-workspace/iteration-1/eval-1-python-project-setup/with_skill/outputs/src/projinit/ui/theme.py
"""Rich theme definitions for consistent CLI styling."""

from rich.theme import Theme
from rich.style import Style

CLI_THEME = Theme(
    {
        "primary": Style(color="cyan"),
        "secondary": Style(color="magenta"),
        "success": Style(color="green", bold=True),
        "warning": Style(color="yellow"),
        "error": Style(color="red", bold=True),
        "info": Style(color="blue"),
        "dim": Style(dim=True),
        "heading": Style(color="cyan", bold=True, underline=True),
        "accent": Style(color="bright_magenta"),
        "code": Style(color="bright_black", bgcolor="grey11"),
    }
)

PROMPT_STYLE = [
    ("qmark", "fg:cyan bold"),
    ("question", "bold"),
    ("answer", "fg:cyan"),
    ("pointer", "fg:cyan bold"),
    ("highlighted", "fg:cyan bold"),
    ("selected", "fg:cyan"),
]

File: .claude/skills/cli-builder-workspace/iteration-2/eval-0-nodejs-csv-converter/with_skill/outputs/src/ui/theme.ts
import chalk, { type ChalkInstance } from 'chalk';

type Theme = {
  readonly primary: ChalkInstance;
  readonly secondary: ChalkInstance;
  readonly success: ChalkInstance;
  readonly warning: ChalkInstance;
  readonly error: ChalkInstance;
  readonly info: ChalkInstance;
  readonly dim: ChalkInstance;
  readonly bold: ChalkInstance;
  readonly heading: ChalkInstance;
  readonly code: ChalkInstance;
  readonly link: ChalkInstance;
};

export const theme: Theme = {
  primary: chalk.cyan,
  secondary: chalk.magenta,
  success: chalk.green,
  warning: chalk.yellow,
  error: chalk.red,
  info: chalk.blue,
  dim: chalk.dim,
  bold: chalk.bold,
  heading: chalk.cyan.bold.underline,
  code: chalk.gray,
  link: chalk.blue.underline,
};

export const colorize = (fn: ChalkInstance, text: string): string =>
  process.env.NO_COLOR ? text : fn(text);
